### PR TITLE
app-builder 7.x | update app builder for 7.3 release | LRDOCS-8181

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-8181

### CODE REVIEW

**Comment by rbohl** [comment]
@matheus-xavier  I added a commit to talk about assigning the task to a user when it's accessed via the notification.

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md [hunk 1]
```diff
@@ -18,17 +18,20 @@ Why use App Builder when the [Liferay Forms](../../process-automation/forms/user
...
 
 ### App Builder Objects
 
-An _Object_ is at the foundation of every App Builder application. It defines the information and types of information that you want to the application to collect and interact with. For example, a Guestbook object would contain a _Name_ field and a _Message_ field, at least. There are two kinds of objects in App Builder:
+An _Object_ is at the foundation of every App Builder application. It defines the information you want the application to collect and interact with. For example, a Guestbook object would contain a _Name_ field and a _Message_ field, at least. There are two kinds of objects in App Builder:
 
+<!-- remove mention of native objects as they won't be here for 7.3 -->
 * _Custom Objects_ are created using a drag-and-drop Form Builder interface.
 * _Native Objects_ are pre-defined sets of data fields that can be used out of the box or extended to suit your needs.
```
Comment (jeyvison) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/app-builder-overview.md]: Remove references to Native Objects

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-a-standard-application.md [hunk 2]
```diff
@@ -13,13 +13,15 @@ Here we'll create two simple time off management applications with App Builder,
 
 Create the Time Off Request custom object:
 
-1. Navigate to Control Panel &rarr; App Builder &rarr; Custom Objects, and click the add button.
+1. Open the Applications Menu (![Applications Menu](../../images/icon-applications-menu.png)) and navigate to Control Panel &rarr; App Builder &rarr; Custom Objects.
```
Comment (jeyvison) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-a-standard-application.md:16]: Custom Objects portlet was renamed to Objects

**Comment by jeyvison** [comment]
Hi @rbohl . Please check above my cooments. I dont know much about the workflow app creation. maybe we should talk with @rafaprax to see if he can help :)

**Comment by rafaprax** [comment]
Hi @rbohl, the documentation looks good to me.

I have one doubt. Do we need to specify that the Workflow Powered Apps is DXP only feature?

Thx

**Comment by rbohl** [comment]
> Hi @rbohl, the documentation looks good to me.
> 
> I have one doubt. Do we need to specify that the Workflow Powered Apps is DXP only feature?
> 
> Thx

@rafaprax 

I can add it to this note: https://github.com/matheus-xavier/liferay-learn/pull/2/files#diff-032532951886ee1d75c001fc7f988c4a1ae496d547454bf025695db0798d1511R85-R87

I also have a [line at the top of the workflow-powered applications article](https://github.com/matheus-xavier/liferay-learn/pull/2/files#diff-b2dbef05e5431cb807a5d2ccfd16ff74c4b30aa5ae4e5260d0041690756c4ba6R3)

like this:

> Subscribers

which is how we are calling out the DXP-only articles on Liferay Learn. Do you think that's enough?

**Comment by rafaprax** [comment]
@rbohl It is enough. Thx :)

**Comment by rbohl** [comment]
Superseded by https://github.com/sez11a/liferay-learn/pull/241

**Pull request closed without merging**